Add types folder and remove any type

kevohm opened the pull request:
Moved typing to the types folder.
Added types folder.

egeuysall commented:
Hello, I currently don’t have time to check for changes because I’m not home. I will check them once I get back.

egeuysall commented:
Hello Kevin,

I'm amazed by what you've done. However, today I edited most of the files, so they might not align with yours. Can you check that?

I'm very sorry for that, I was just trying to finish the project.

Have a great day, thank you.

Regards,
Ege

kevohm commented:
Hello. I will look at it tomorrow. I am currently held up a bit.

egeuysall reviewed (changes requested):
Hello, there are some errors. For example, the current header is different from what you’ve done. Also, some of the components like the Marquee and what people are saying sections are not built. Please modify those and we can merge the pull request. Thank you for such an amazing help!

kevohm commented:
Hello, hope you are fine. Where can I view the current version?

kevohm commented:
Hello. Can you run it locally and see if there are any issues?

egeuysall commented:
> Hello. Can you run it locally and see if there are any issues?

Hello, of course. I didn’t see your response.

egeuysall commented:
> Hello. Can you run it locally and see if there are any issues?

Hello, I checked and some features are still missing. For example, the mobile header is not the same in yours and some component placements are wrong and not identical to the main project. Also, the corner rounding feature is missing and pnpm build returns errors.

kevohm commented:
Looking into it.

kevohm commented:
![Screenshot 2025-04-11 120503](https://github.com/user-attachments/assets/eae9c76c-0530-4b11-86ba-bfd13ad68335)
These are the linting issues. Should I fix them?
I have cross-checked the current version with the live site, but I can't find differences.  Am I missing something?

egeuysall commented:
> ![Screenshot 2025-04-11 120503](https://private-user-images.githubusercontent.com/58782210/432684609-eae9c76c-0530-4b11-86ba-bfd13ad68335.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNzkwNDAsIm5iZiI6MTc0NDM3ODc0MCwicGF0aCI6Ii81ODc4MjIxMC80MzI2ODQ2MDktZWFlOWM3NmMtMDUzMC00YjExLTg2YmEtYmZkMTNhZDY4MzM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDEzMzkwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiMjhiMGJhZjI5ODQzM2JiNjE4ZDAzOGY2ZDQ4OTM1MDc2ZmVlZmY5NzdhYTQ1M2ZhNjhlMjEyNGM4ZDFkZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-Z0mE0HyUXdguYyU1ShoTeUzHggXJ50cwW6HbSbvBxw) These are the linting issues. Should I fix them? I have cross-checked the current version with the live site, but I can't find differences. Am I missing something?

Hello, yeah I suggest fixing them because the build doesn’t work without solving them. You might be looking in a wrong live website, the live website’s link is [links.egeuysal.com](https://www.links.egeuysal.com/) If you look carefully, you can see that the mobile navigation bar animations are missing, button corner rounding features are missing in the create page and the what people are saying doesn’t exist in yours.

kevohm commented:
## Fixed animations and linting issues
I fixed the suggested issues. However, I still get an error when building the app. It may be related to the .env file.

PS: You may have to check if the globe is rendered on your end. I made no breaking changes to your code on this. I added useRef for Phi and width variables.

## Screenshots
- Globe view
![Screenshot 2025-04-13 135235](https://github.com/user-attachments/assets/aec0321e-4fd2-4b7e-a223-e2c472edecd4)

- Error on build
![Screenshot 2025-04-13 134752](https://github.com/user-attachments/assets/40ed1128-8c24-46c5-b7c0-dc7e05fe959a)

- Button rounding feature
![Screenshot 2025-04-13 134219](https://github.com/user-attachments/assets/2f750f25-9477-40ee-840c-7a0897eb3b56)

- Menu animation
![Screenshot 2025-04-13 134301](https://github.com/user-attachments/assets/d05a8bb2-974d-4e23-ae60-6ca285846d43)

- Testimonials
![Screenshot 2025-04-13 134436](https://github.com/user-attachments/assets/9ae59f26-330c-471c-b9a8-f84fe9e9c00f)

egeuysall commented:
Amazing, thank you! I will merge it and let’s see! Thank you for the amazing work!

egeuysall reviewed (commented):
# Well done!
All animations applied.

egeuysall reviewed (commented):
# Removed unused variables.

Nice, I knew that there were a lot of unused variables. Thank you!

egeuysall reviewed (commented):
Thank you for using pnpm! It’s much more performant!

egeuysall reviewed (commented):
# Thank you!
I will update the change log content soon as it’s quite invalid

kevohm commented:
It was nice working with you. I love what you are building and hope we work together again. Happy coding : )
